src/app/components/home/RewardsPanel.tsx

```diff
@@ -205,15 +205,17 @@ const NetworkFeeSection = () => {
 
   const changeShouldLedgerSign = useChangeShouldLedgerSign();
   const addTransaction = useTransactionAdder();
+
+  const DAY_LIMIT = 49;
```

Comment:
Could you use the constant variable `BATCH_SIZE`? @Chunn 
Did you confirm that the UI claims the same amount same as the showing amount?

Reply:
@octocat512 I've used `DAY_LIMIT` instead of `BATCH_SIZE` based on this comment https://github.com/balancednetwork/balanced-network-interface/issues/833#issue-1092830176
> Did you confirm that the UI claims the same amount same as the showing amount?

 I don't have an account on mainnet. So, I've asked @rise-techiast  to check for me. It seems it's OK

Reply:
We've discussed and agreed to use `BATCH_SIZE` only